backend/app/services/web_scraper/scraper_config.py

```
@@ -0,0 +1,87 @@
+                reverse=True,
+            )
+        )
+        logger.info(
+            f"[WebScraperConfig] Normalized config keys: {list(normalized_config.keys())}"
+        )
+        return normalized_config
+
+    except json.JSONDecodeError as e:
+        logger.error(f"[WebScraperConfig] Failed to parse WEB_SCRAPER_SITE_CONFIG: {e}")
+        return {}
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)